CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Codeowners for IREE Github Repository.
+# Refer to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
```

Comment:
We could move this to .github/ instead of the root. I'd prefer it there, since the root has many files already.